src/patternfly/components/Expandable/expandable.scss

```diff
@@ -0,0 +1,84 @@
+@import "../../patternfly-imports";
+
+.pf-c-expandable {
+  // Expandable
+  --pf-c-expandable--Width: #{pf-size-prem(450px)};
+  --pf-c-expandable--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
```

Comment:
If this should always have a white background (assuming it would always be a light theme), you can use `--pf-global--BackgroundColor--100` as the background color and add `@include pf-t-light;`